apps/tup-cms/Dockerfile

```diff
@@ -8,3 +8,6 @@ COPY apps/tup-cms/src/taccsite_cms /code/taccsite_cms
 
 COPY dist/apps/tup-ui/imports.html /code/taccsite_cms/templates/imports.html
 COPY dist/apps/tup-ui/assets/ /code/taccsite_cms/static/assets/
+
+COPY dist/apps/tup-cms-react/assets /code/taccsite_cms/static/cms-react/assets
+COPY /dist/apps/tup-cms-react/imports.html /code/taccsite_cms/templates/assets_custom_delayed.html
```

Comment:
Hm… this means that if `tup-cms` has a manually-created `assets_custom_delayed.html` (copied via `COPY apps/tup-cms/src/taccsite_cms /code/taccsite_cms`), it will be overwritten. I think I will create—as a warning to others (or future me)—a manual `assets_custom_delayed.html` with a comment that explains such. Opinions? Concerns?

Reply:
Yeah, that sounds like a good plan. I like the idea of having a designated "override" template that allows an image built on core-cms to insert scripts/markup.

Reply:
Okay. I added it in #165.